src/coreclr/src/inc/corinfo.h

```diff
@@ -208,11 +208,11 @@ TODO: Talk about initializing strutures before use
 //
 //////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-constexpr GUID JITEEVersionIdentifier = { /* 062114d0-bd20-483f-8a3e-c4ee39706ae8 */
-    0x062114d0,
-    0xbd20,
-    0x483f,
-    {0x8a, 0x3e, 0xc4, 0xee, 0x39, 0x70, 0x6a, 0xe8}
+constexpr GUID JITEEVersionIdentifier = { /* a0184d06-a562-43f6-94ad-44627db87310 */
```

Comment:
Would you mind deleting unused `CORINFO_HELP_INTERNALTHROW` and `CORINFO_HELP_SEC_UNMGDCODE_EXCPT` while you are bumping the JIT/EE interface GUID?

(I had it on my cleanup list, but I did want to bump the GUID just to delete a few helpers.)